kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/BigQuerySinkTask.java

```diff
@@ -155,6 +155,12 @@ private RowToInsert getRecordRow(SinkRecord record) {
       convertedRecord = FieldNameSanitizer.replaceInvalidKeys(convertedRecord);
     }
 
+    if (config.useTimestampPartitioning()) {
+      if (!convertedRecord.containsKey(config.getTimestampPartitionFieldName())) {
```

Comment:
When/how would this happen? Looks like the first if statement accounts for the field name being non empty. Of course this doesn't ensure the record containing the field, but still wanted to know this.